scripts/setup-plan.sh

```
@@ -0,0 +1,44 @@
+    case "$arg" in
+        --json) JSON_MODE=true ;;
+        --help|-h) echo "Usage: $0 [--json]"; exit 0 ;;
+    esac
+done
+
+# Source common functions
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+source "$SCRIPT_DIR/common.sh"
+
+# Get all paths
+eval $(get_feature_paths)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)